.github/workflows/docker-build.yaml

```diff
@@ -69,10 +71,14 @@ jobs:
               docker/Dockerfile.base
             sed -i 's/balenalib\/raspberrypi3/balenalib\/raspberry-pi2/' \
               docker/Dockerfile.viewer
+            sed -i 's/\barmv7hf\b/rpi/' \
```

Comment:
This shouldn't be necessary since you're using the base image (which we patch above).

Reply:
Same comment as with the other thread.